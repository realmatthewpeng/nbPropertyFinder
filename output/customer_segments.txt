Total number of lines found: 10

display(data.describe())
display(samples - data.mean().round())
samples_percentage['Total'] = samples_percentage.sum(axis=1)
samples_w_total['Total'] = samples_w_total.sum(axis=1)
samples_w_total = samples.copy()
samples_percentage
new_data = data.copy()
y_all = data[column_to_study]
X_all = new_data.drop(column_to_study, axis=1)
df_preds = pd.concat([df_diffs, pd.Series(sample_preds, name='PREDICTION')], axis=1)

Incorrect Lines:

df_preds = pd.concat([df_diffs, pd.Series(sample_preds, name='PREDICTION')], axis=1)

Precision = 0.9
Recall = 0.375

data.drop(['Region', 'Channel'], axis = 1, inplace = True)
print "Wholesale customers dataset has {} samples with {} features each.".format(*data.shape)
samples = pd.DataFrame(data.loc[indices], columns = data.keys()).reset_index(drop = True)
display(samples)
display(samples - data.median().round())
samples_percentage = samples.loc[:,"Fresh":"Delicatessen"].div(samples_w_total['Total'], axis=0) * 100
log_data = np.log(data)
log_samples = np.log(samples)
display(log_samples)
Q1 = np.percentile(log_data[feature], 25)
Q3 = np.percentile(log_data[feature], 75)
df_outliers = log_data[~((log_data[feature] >= Q1 - step) & (log_data[feature] <= Q3 + step))]
display(df_outliers)
good_data = log_data.drop(log_data.index[outliers]).reset_index(drop = True)
true_centers = np.exp(log_centers)
