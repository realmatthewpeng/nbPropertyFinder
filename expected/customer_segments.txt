data.drop(['Region', 'Channel'], axis = 1, inplace = True)
print "Wholesale customers dataset has {} samples with {} features each.".format(*data.shape)
display(data.describe())
samples = pd.DataFrame(data.loc[indices], columns = data.keys()).reset_index(drop = True)
display(samples)
display(samples - data.mean().round())
display(samples - data.median().round())
samples_w_total = samples.copy()
samples_w_total['Total'] = samples_w_total.sum(axis=1)
samples_percentage = samples.loc[:,"Fresh":"Delicatessen"].div(samples_w_total['Total'], axis=0) * 100
samples_percentage['Total'] = samples_percentage.sum(axis=1)
samples_percentage
new_data = data.copy()
X_all = new_data.drop(column_to_study, axis=1)
y_all = data[column_to_study]
log_data = np.log(data)
log_samples = np.log(samples)
display(log_samples)
Q1 = np.percentile(log_data[feature], 25)
Q3 = np.percentile(log_data[feature], 75)
df_outliers = log_data[~((log_data[feature] >= Q1 - step) & (log_data[feature] <= Q3 + step))]
display(df_outliers)
good_data = log_data.drop(log_data.index[outliers]).reset_index(drop = True)
true_centers = np.exp(log_centers)